### PR TITLE
Minor typo correction

### PR DESCRIPTION
**Pull request by Andre601**

### CODE REVIEW

File: pages/legal/privacy.vue [hunk 1]
```diff
@@ -144,7 +144,7 @@
       </p>
 
       <p>
-        Modrinth makes use external ads providers. These include BuySellAds. You
+        Modrinth makes use of external ads providers. These include BuySellAds. You
```
Comment (wafflecoffee) [inline pages/legal/privacy.vue:147]: Lint failed

**Review by wafflecoffee — CHANGES_REQUESTED** [review]

File: pages/legal/privacy.vue (hunk 1 above)
Comment (Andre601) [reply pages/legal/privacy.vue:147]: Well...
```
Error:   1[47](https://github.com/modrinth/knossos/runs/8274622938?check_suite_focus=true#step:7:48):80  error  Delete `·You`  prettier/prettier
Error:   148:8   error  Insert `·You`  prettier/prettier
```
Not sure what this should tell me...

Comment (wafflecoffee) [reply pages/legal/privacy.vue:147]: The "You" needs to be on the next line

Comment (Andre601) [reply pages/legal/privacy.vue]: fixed

**Review by wafflecoffee — APPROVED** [review]

**Pull request merged**